__tests__/resources/acts-1-20.aligned.oldformat.usfm

```diff
@@ -1,86 +1,86 @@
 \id ACT EN_ULT en_English_ltr Wed Oct 10 2018 11:55:23 GMT-0400 (EDT) tc
 \c 1
 \v 20 "
-\zaln-s | x-strong="G10630" x-lemma="γάρ" x-morph="Gr,CC,,,,,,,," x-occurrence="1" x-occurrences="1" x-content="γὰρ"
+\zaln-s |x-strong="G10630" x-lemma="γάρ" x-morph="Gr,CC,,,,,,,," x-occurrence="1" x-occurrences="1" x-content="γὰρ"
```

Comment:
We should leave the spaces in this file.  This is to verify that we can still import the old file format.